scripts/compose.py

```diff
@@ -778,10 +778,28 @@ def _content(self):
         )
 
 
+class AgentJavaSpring(Service):
+    SERVICE_PORT = 8090
+
+    def _content(self):
+        return dict(
```

Comment:
can we get a healthcheck on this service to avoid flakiness in tests running before this is actually ready?